### PR TITLE
sections: fix small things

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

File: t.formats/elf/tiny [hunk 1]
```diff
@@ -20,10 +20,11 @@ run_test
 # S 0x00000000 0x00010000 0x00020040 0x00020040 phdr0 r--
 # S 0x00000000 0x00010000 0x0000002d 0x0000002d ehdr rwx
 NAME='ELF: corkami tiny.elf - sections'
+BROKEN=1
 FILE=../../bins/elf/tiny.elf
 ARGS=''
 CMDS="S"
 EXPECT='[00] * 0x00000000 rwx va=0x00010000 sz=0x002d vsz=0x002d uphdr
-[01] * 0x00000000 rwx va=0x00000000 sz=0x002d vsz=0x002d ehdr
+[01] . 0x00000000 rw- va=0x00000000 sz=0x002d vsz=0x002d ehdr
```
Comment (ret2libc) [inline t.formats/elf/tiny]: the offset when you open the file is 0x00010020 so i expect that we are in the uphdr section, not ehdr.

Comment (ret2libc) [inline t.formats/elf/tiny]: moreover ehdr is not even really mapped in the virtual space of the process, so there shouldn't be any executable flag imo

File: t.formats/pe/hellocxx [hunk 2]
```diff
@@ -15,12 +15,14 @@ EXPECT='0x401280
...
 
 NAME='PE: hello cxx - sections'
+BROKEN=1
 FILE=../../bins/pe/hellocxx-mingw32.exe
 ARGS=''
 CMDS="S*~^S"
 EXPECT='S 0x00000400 0x00401000 0x0003fc00 0x0003fb80 .text r-x
 S 0x00040000 0x00441000 0x00000200 0x00000168 .data rw-
 S 0x00040200 0x00442000 0x00002c00 0x00002ba0 .rdata r--
+S 0x00000000 0x00445000 0x00000000 0x00001e10 .bss rw-
```
Comment (ret2libc) [inline t.formats/pe/hellocxx]: When i list the sections with ```S``` i cannot see the .bss section, even though i think it should be mapped in the buffer and displayed as a list of 0, not ff (ff seems to be used for unmapped regions)

File: t.io/ff [hunk 3]
```diff
@@ -24,3 +24,12 @@ EXPECT='00000000000000000000
 00000000000000000000
 '
 run_test
+
+NAME=".bss no FF but 0"
```
Comment (ret2libc) [inline t.io/ff]: Same as above. I expect the bss section to be mapped even in non-debug mode.
What do you think?

File: t.formats/pe/hellocxx (hunk 2 above)
Comment (radare) [inline t.formats/pe/hellocxx]: The bss section is allocated by the loader, it should be mapped from malloc:// maybe

that’s why is not loaded by S

Comment (ret2libc) [inline t.formats/pe/hellocxx]: Well when we map things we are actually trying to do the work of the loader, so we should also have the bss in some way. If you look at IDA (sorry to mention it :P ) you can clearly see that some regions are bss

**Pull request closed without merging**